org/responses.py

```diff
@@ -22,3 +22,34 @@
 user_already_present_409 = {
   "message": "Already a member of organization"
 }
+
+list_permission_set_200 = [
+  {
```

Comment:
This response has been deleted from the `dev` branch and should not be added by this PR.

Reply:
ok